src/features/analytics/components/IndicatorComponent/IndicatorComponent.css

```diff
@@ -0,0 +1,166 @@
+.absolute-component,
+.marker-list,
+.single-component,
+.list-component {
+  position: relative;
+  border-radius: 10px;
+  width: 100%;
+  height: auto;
+  box-shadow: 0 0 43px rgb(24 14 54 / 20%);
+  background: #fff;
+}
+
+.absolute-component {
+  padding: 32px 32px 42px;
+}
+
+.absolute-component__sum {
+  border-bottom: 1px solid #c2c2c2;
+  padding-bottom: 26px;
+}
+
+.absolute-component__values {
+  display: flex;
+  margin-top: 24px;
+}
+
+.absolute-component__values div.absolute-component__value:first-child {
+  margin-right: 60px;
+}
+
+.marker-list {
+  padding: 106px 32px 32px;
+}
+
+.marker-list__head {
+  margin-bottom: 24.17px;
+  font-weight: 500;
+  font-size: 26px;
+}
+
+.marker-list__info {
+  display: flex;
+}
+
+.marker-list__info-labels {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+}
+
+.marker-list__info-labels div:nth-last-child(n+2) {
+  margin-bottom: 32.33px;
+}
+
+.marker-list__info-values {
+  display: flex;
+  align-items: flex-end;
+  flex-direction: column;
+  width: 50%;
+}
+
+.marker-list__info-values div:nth-last-child(n+2) {
```

Comment:
try not to use tags in styles. 
If you want to change the tag in the component, the style will not be applied to it